spec/jobs/reports/irs_fraud_metrics_report_spec.rb

```diff
@@ -90,6 +90,7 @@
     it 'sends out a report to just to team data' do
```

Comment:
what is the expected behavior here? Don't we want it to always put the internal recipients in the BCC line? @ambuj-neupane-tts